### PR TITLE
Ria 2767 api docs

### PR DESCRIPTION
**Pull request by Lee365**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-2767

### Change description ###
RIA-2767  Swagger Docs publish

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,5 @@
-#Fri Aug 03 13:37:52 BST 2018
-distributionBase=GRADLE_USER_HOME
-distributionPath=wrapper/dists
-zipStoreBase=GRADLE_USER_HOME
-zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip
```
Comment (timja) [inline gradle/wrapper/gradle-wrapper.properties:3]: why is this 4.10.2? this should either be updated to 6.x here or moved to a separate PR.

The rest of the 6.x changes are in this PR as far as I can tell but you've force pushed and messed with the commit history for some reason so I can't tell

File: infrastructure/main.tf [hunk 2]
```diff
@@ -7,125 +7,125 @@ locals {
...
 data "azurerm_key_vault_secret" "test_law_firm_a_username" {
   name      = "test-law-firm-a-username"
-  vault_uri = "${data.azurerm_key_vault.ia_key_vault.vault_uri}"
+  vault_uri = data.azurerm_key_vault.ia_key_vault.vault_uri
 }
 
 data "azurerm_key_vault_secret" "test_law_firm_a_password" {
   name      = "test-law-firm-a-password"
-  vault_uri = "${data.azurerm_key_vault.ia_key_vault.vault_uri}"
+  vault_uri = data.azurerm_key_vault.ia_key_vault.vault_uri
```
Comment (timja) [inline infrastructure/main.tf:51]: the changes in this file won't work, you aren't on the terraform version required for this

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (Lee365) [reply gradle/wrapper/gradle-wrapper.properties:3]: https://tools.hmcts.net/jira/browse/RIA-2951
is the story to upgrade gradle version.
For now I would like to push with these changes and I have informed Ravi

Comment (timja) [reply gradle/wrapper/gradle-wrapper.properties:3]: You should revert the gradle wrapper changes then

Comment (Lee365) [reply gradle/wrapper/gradle-wrapper.properties:3]: gradle version reverted to 4.10.2

File: infrastructure/main.tf (hunk 2 above)
Comment (Lee365) [reply infrastructure/main.tf:51]: Tim did

**Pull request merged**